python/taichi/lang/matrix.py

```diff
@@ -590,6 +584,47 @@ def rotation2d(alpha):
         return ti.Matrix([[ti.cos(alpha), -ti.sin(alpha)],
                           [ti.sin(alpha), ti.cos(alpha)]])
 
+    @staticmethod
+    def var(n, m, dt, shape):
+        import taichi as ti
+        return ti.Matrix(n=n, m=m, dt=dt, shape=shape)
+
+    @staticmethod
+    def rows(rows):
+        import taichi as ti
+        mat = ti.Matrix()
```

Comment:
```suggestion
        mat = ti.Matrix().empty()
```
since it's previous content won't be used, just a placeholder.